lighthouse-cli/test/fixtures/perf/perf-budgets/load-things.html

```diff
@@ -30,7 +30,7 @@
         // Load file as 'other' resource.
         const response = await fetch('http://localhost:10503/preload.html');
         const ab = await response.arrayBuffer();
-        document.querySelector('#corsfont').textContent = ab.length;
```

Comment:
this change is just about text and doesn't matter at all right?

Reply:
idk, would the font be loaded if there were no text?

Reply:
oh, oh I see what I'm missing now sorry :) 

1. `new ArrayBuffer().length === undefined`, not just a semantic difference like in node
2. `el.textContent = undefined` clears all textContent, and does not do some sort of string coercion to `"undefined"`

Reply:
> el.textContent = undefined clears all textContent, and does not do some sort of string coercion to "undefined"

just to be clear, while the actual behavior is unintended and pointless, the intended behavior was also pointless. There's no reason to add this to the page at all, just making sure (poorly, it turns out) the thing is actually loaded and used :)